templates/projects/web/_gulpfile.js

```diff
@@ -1,19 +1,32 @@
-var gulp = require('gulp');
-var bower = require('gulp-bower');
-var del = require('del');
-var project = require('./project.json');
-var lib = project.webroot + '/lib';
+/// <binding Clean='clean' />
 
-gulp.task('default', ['bower:install'], function () {
-    return;
-});
+var gulp = require("gulp"),
+  rimraf = require("rimraf"),
+  fs = require("fs");
+
+eval("var project = " + fs.readFileSync("./project.json"));
```

Comment:
Can this line be rewritten to not use eval? We should not ship eval to users in template as of 2015 I think:
```JavaScript
var project = require('./project.json');
var paths = {
  bower: "./bower_components/",
  lib: "./" + project.webroot + "/lib/"
};
```